docs/docs/build/tokenproviders.mdx

```diff
@@ -3,6 +3,6 @@ title: "Tutorial: Writing a TokenProvider with an Azure Function"
 sidebar_position: 6
 ---
 
-In the [Fluid Framework](https://fluidframework.com/), TokenProviders are responsible for creating and signing tokens that the `@fluidframework/azure-client` uses to make requests to the Azure Fluid Relay. Each Fluid service must implement a custom TokenProvider based on the particular service's authentication and security considerations.
```

Comment:
Note: users are already on our website - I don't see a need to link back to the homepage from here.